src/Fixer/Alias/PowToExponentiationFixer.php

```diff
@@ -209,13 +209,11 @@ private function isParenthesisNeeded(Tokens $tokens, int $argumentStartIndex, in
      */
     private function getAllowedKinds(): array
     {
-        return array_merge(
-            [
-                T_DNUMBER, T_LNUMBER, T_VARIABLE, T_STRING, T_CONSTANT_ENCAPSED_STRING, T_DOUBLE_CAST,
-                T_INT_CAST, T_INC, T_DEC, T_NS_SEPARATOR, T_WHITESPACE, T_DOUBLE_COLON, T_LINE, T_COMMENT, T_DOC_COMMENT,
-                CT::T_NAMESPACE_OPERATOR,
-            ],
-            Token::getObjectOperatorKinds()
-        );
+        return [
```

Comment:
`array_merge` should be fixed using a rule - it is the same since PHP 8.1 https://3v4l.org/mKbr7 - the reason to use spread operator is it can be more optimized by php-src

Reply:
I don't understand your comment - what are you referring to, @mvorisek?

Reply:
instead of this manual PR fix it by a new rule

Reply:
You never know if it was manual work 😉.